defaults/main.yml

```diff
@@ -66,8 +64,8 @@ oauth_secret: ~
 oauth_logo_url: 'https://status.app/assets/favicon/default.png'
 oauth_provider_scopes:
   github: 'user:email read:org'
-  keycloak: 'openid'
-oauth_scope: '{{ oauth_provider_scopes.get("oauth_provider", "") }}'
+  keycloak-oidc: 'openid'
```

Comment:
This change will break all the current playbook that use keycloak as provider.
From what i know : Prometheus, Vault, Airbyte test.